resources/views/dashboard/admin/live.blade.php

```diff
@@ -0,0 +1,38 @@
+@extends('layouts.dashboard')
+
+@section('title')
+Live Event Information
+@endsection
+
+@section('content')
+@include('inc.header', ['title' => 'Live Event Information'])
+
+<div class="container">
+    {{ html()->form()->route('saveLiveEventInfo')->open() }}
+        @csrf
+        <div class="form-group">
+            <label for="event_title" class="control-label">Live Event Name</label>
+            {{ html()->text('event_title', $liveInfo->event_title)->placeholder('Required')->class(['form-control']) }}
```

Comment:
We should probably change this to "Leave blank for no info"

Reply:
Done!